src/test/java/uk/gov/dhsc/htbhf/steps/Hooks.java

```diff
@@ -1,16 +1,25 @@
 package uk.gov.dhsc.htbhf.steps;
 
 import io.cucumber.java.After;
+import io.cucumber.java.Before;
 
 public class Hooks extends BaseSteps {
+
+    @Before
+    public void initDriver() {
+        if ("browserstack".equals(activeProfile)) {
```

Comment:
Suggest pulling the check into a method

Reply:
Yep, I'll do it in the next PR so I don't dismiss any reviews.